src/lib/utils/updaterUtils.js

```diff
@@ -0,0 +1,46 @@
+import { isElement } from "lodash";
+
+export function getGraphDiv(gdID) {
+    // see this link for more information https://stackoverflow.com/a/34002028
+    let graphDiv = document?.querySelectorAll('div[id*="' + gdID + '"][class*="dash-graph"]');
+    if (graphDiv.length > 1) {
```

Comment:
Is it possible to distinguish between multiple matches versus multiple unique IDs? If I clone a graph and want the update to apply to the clone as well as the original, the clone would have the same ID.